packages/wrangler/src/assets.ts

```diff
@@ -374,52 +363,40 @@ export function processExperimentalAssetsArg(
  */
 export function validateAssetsArgsAndConfig(
 	args:
-		| Pick<
-				StartDevOptions,
-				"legacyAssets" | "site" | "experimentalAssets" | "script"
-		  >
-		| Pick<
-				DeployArgs,
-				"legacyAssets" | "site" | "experimentalAssets" | "script"
-		  >,
+		| Pick<StartDevOptions, "legacyAssets" | "site" | "assets" | "script">
+		| Pick<DeployArgs, "legacyAssets" | "site" | "assets" | "script">,
 	config: Config
 ) {
 	/*
 	 * - `config.legacy_assets` conflates `legacy_assets` and `assets`
 	 * - `args.legacyAssets` conflates `legacy-assets` and `assets`
 	 */
 	if (
-		(args.experimentalAssets || config.experimental_assets) &&
+		(args.assets || config.assets) &&
 		(args.legacyAssets || config.legacy_assets)
 	) {
 		throw new UserError(
-			"Cannot use Experimental Assets and Legacy Assets in the same Worker.\n" +
-				"Please remove either the `site` or `experimental_assets` field from your configuration file."
+			"Cannot use assets and legacy assets in the same Worker.\n" +
+				"Please remove either the `site` or `assets` field from your configuration file."
```

Comment:
@emily-shen  I just realised that this call to action is not 100% accurate, since `assets` or `legacy_assets` could have come from either config or args :/. So we might want to rethink this a bit.

Reply:
What is the official name for this whole thing anyway 😅 ? Workers Assets? Workers + Assets? Just wondering about the capitalisation

Reply:
judging from https://bed854e4.cloudflare-docs-7ou.pages.dev/workers/configuration/assets/, it just `assets`

Reply:
in that case, here is a nit 😅 

`"Cannot use Assets and Legacy Assets in the same Worker.\n"`

Reply:
why capitalized?

Reply:
nvm I see everything is lower case - not a big deal, please ignore!